CRM/Profile/Selector/Listings.php

```diff
@@ -598,7 +598,7 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL, $ex
           $typeId = substr($name, 0, -4) . "-website_type_id";
           $typeName = CRM_Core_PseudoConstant::getLabel('CRM_Core_DAO_Website', 'website_type_id', $result->$typeId);
           if ($typeName) {
-            $row[] = "<a href=\"$url\">{$result->$name} (${typeName})</a>";
+            $row[] = "<a href=\"$url\">{$result->$name} ({$typeName})</a>";
```

Comment:
Technically the curly braces are unnecessary and this can just be 
```suggestion
            $row[] = "<a href=\"$url\">{$result->$name} ($typeName)</a>";
```